shared/utils/dates.ts

```diff
@@ -0,0 +1,6 @@
+import { format, parse } from 'date-fns'
+import { MYINFO_DATE_FORMAT, DATE_PARSE_FORMAT } from '../constants'
+
+export const formatMyinfoDate = (value: string) => {
```

Comment:
Very big nit but should it be `formatMyInfoDate`? Cause we do capitalise `Info` right hahah

Reply:
haha so ACTUALLY I had the same thought, then I googled to check. Turns out it's actually [Myinfo](https://api.singpass.gov.sg/library/myinfo/business/implementation-myinfo-data) instead of MyInfo lolol